addons/web/static/src/views/pivot/pivot_model.js

```diff
@@ -1220,7 +1220,10 @@ export class PivotModel extends Model {
                     rowIndex === 0
                         ? undefined
                         : fields[colGroupBys[rowIndex - 1].split(":")[0]].string,
-                title: group.labels[group.labels.length - 1] || _t("Total"),
+                title:
```

Comment:
```suggestion
                title: group.labels.length ? group.labels[group.labels.length - 1] : _t("Total"),
```

Reply:
a bit simpler but apart from that I agree with the fix. Thank you PRO

Reply:
Updated :)